thread-context/src/main/java/com/github/threadcontext/CombinedContextSupplier.java

```diff
@@ -12,7 +13,9 @@ public CombinedContextSupplier(Iterable<Supplier<Context>> suppliers) {
 
     public Context get() {
         Context result = null;
-        for (Supplier<Context> supplier : suppliers) {
```

Comment:
Why was this throwing a concurrent modification exception?

Reply:
I'm worried that this might not actually fix the issue.

Reply:
For example, if the exception is caused because one thread is iterating over the suppliers while another thread is inserting into the suppliers, this won't fix anything, and we instead need to either use synchronization, or a thread-safe collection.